examples/decode_market_example/main.py

```
@@ -25,52 +25,47 @@ def main():
+                    print(f"    [Action]      {details['action_id']}")
+                    print(f"    [Market Type] {details['type'].upper()}")
+                    
+                    # Format thresholds for readability
+                    thresholds_str = ", ".join(details['thresholds'])
+                    print(f"    [Thresholds]  {thresholds_str}")
+                    print(f"    [Stream ID]   {details['stream_id']}")
+                    print()
 
-            except Exception as e:
-                print(f"  [!] Error processing market {market_id}: {e}")
+                except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)